tests/testthat/test-config.R

```diff
@@ -233,31 +262,61 @@ test_that(
 )
 
 test_that(
-  "read_predevals_config fails, invalid eval window n_last",
+  "read_predevals_config fails, invalid eval set n_last",
```

Comment:
unhelpful diff -- these tests are minor updates of existing tests.  the relevant diffs show up below.